src/traces/bar/helpers.js

```diff
@@ -64,3 +64,12 @@ exports.getValue = function(arrayOrScalar, index) {
     else if(index < arrayOrScalar.length) value = arrayOrScalar[index];
     return value;
 };
+
+exports.getLineWidth = function(trace, di) {
```

Comment:
Bars don't have `line.width`, but still calling this `getMarkerLineWidth` would be better.